KinSDK/KinSDKTests/KinAccountTests.swift

```diff
@@ -266,6 +267,91 @@ class KinAccountTests: XCTestCase {
+        }
+
+        do {
+            try account0.sendTransaction(to: account1.publicAddress,
+                                         kin: 5,
+                                         passphrase: passphrase)
+
+            try account1.sendTransaction(to: account1.publicAddress,
+                                         kin: 10,
+                                         passphrase: passphrase)
+        }
+        catch {
+            XCTAssertTrue(false,
+                          "Consecutives transactions with correct passphrase should have succeeded: \(error)")
+        }
+    }
+
+    func test_send_transaction_with_empty_passphrase_after_successful_send() {
+        guard
+            let account0 = account0,
+            let account1 = account1 else {
+                XCTAssertTrue(false, "account creation failed")
+                return
+        }
+
+        do {
+            try account0.sendTransaction(to: account1.publicAddress,
+                                         kin: 5,
+                                         passphrase: passphrase)
+        }
+        catch {
+            XCTAssertTrue(false, "Transaction with correct passphrase should have succeeded: \(error)")
+        }
+
+        do {
+            try account1.sendTransaction(to: account1.publicAddress,
+                                         kin: 10,
+                                         passphrase: "")
+            XCTAssertTrue(false, "Transaction with empty passphrase should have failed.")
+        }
+        catch {
+            let nsError = error as NSError
+            XCTAssertEqual(nsError.domain, "go")
+            XCTAssertEqual(nsError.code, 1)
+            XCTAssertEqual(nsError.localizedDescription, "could not decrypt key with given passphrase")
+        }
+    }
+
+    func test_send_transaction_with_wrong_passphrase_after_successful_send() {
+        guard
+            let account0 = account0,
+            let account1 = account1 else {
+                XCTAssertTrue(false, "account creation failed")
+                return
+        }
+
+        do {
+            try account0.sendTransaction(to: account1.publicAddress,
+                                         kin: 5,
+                                         passphrase: passphrase)
+        }
+        catch {
+            XCTAssertTrue(false, "Transaction with correct passphrase should have succeeded: \(error)")
+        }
+
+        do {
+            try account1.sendTransaction(to: account1.publicAddress,
+                                         kin: 10,
+                                         passphrase: "SomeWrongDumbPassword")
+            XCTAssertTrue(false, "Transaction with wrong passphrase should have failed.")
+        }
+        catch {
+            let nsError = error as NSError
+            XCTAssertEqual(nsError.domain, "go")
+            XCTAssertEqual(nsError.code, 1)
+            XCTAssertEqual(nsError.localizedDescription, "could not decrypt key with given passphrase")
```

Comment:
if geth did actually localize anything in their error messages, this would fail on machines with non us region.